### PR TITLE
Use myst to parse markdown docs, and convert a page from rst to markdown.

### PR DESCRIPTION
**Pull request by skye**

### CODE REVIEW

**Comment by skye** [comment]
The content of profiling.md should be the same as profiling.rst, I can't figure out how to get git to treat it as a rename...

**Comment by shoyer** [comment]
I think git would notice it's the same file if most of the lines match, but you changed the indentation?

File: docs/profiling.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+XLA has some built-in support for profiling on both CPU and GPU. To use XLA's
+profiling features from JAX, set the environment variables
+`TF_CPP_MIN_LOG_LEVEL=0` and `XLA_FLAGS=--xla_hlo_profile`. XLA will log
+profiling information about each computation JAX runs. For example:
+
+```shell
+$ TF_CPP_MIN_LOG_LEVEL=0 XLA_FLAGS=--xla_hlo_profile ipython
+...
+In [1]: from jax import lax
+lax.add
```
Comment (shoyer) [inline docs/profiling.md:22]: this line looks extraneous?

**Review by shoyer — APPROVED** [review]

File: docs/profiling.md (hunk 1 above)
Comment (skye) [reply docs/profiling.md]: Removed.

**Pull request merged**